fuzz/fuzz_targets/from_string_parsing.rs

```diff
@@ -0,0 +1,14 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(s) = std::str::from_utf8(data) {
+        let options = grass::Options::default();
```

Comment:
Should we compare to `dart-sass` as part of fuzzing to ensure the same output?

Reply:
I would do that once panics are resolved in a separate target, but grass already panics on some inputs so those need to be resolved first before continuing.

Reply:
This is planned as something to be done in the future. We have enough failing spec tests that it wouldn't be worthwhile actively fuzzing for differences between the two implementations just yet.

Reply:
@connorskees , i haven't taken a look at the spec tests, but it might still be beneficial to start fuzzing before all spec tests pass, as they might be able to give us a very small input for which the outputs differ (again, haven't looked at the spec tests), allowing to more easily reason about some differences, and fixing them in an isolated fashion